source/plugin/advanced/data/customdata.rst

```diff
@@ -0,0 +1,9 @@
+=======================================
+Implementing Custom Data / Data Holders
+=======================================
+
+Stub: This page shall contain
+
+* Motivation: Why should you want to use the Data API for your custom Data? (Serialization should be interesting)
```

Comment:
Why *shouldn't* you use the DataAPI?

(possibly when indexing is required, or other stuff better served by a database.)